markets/templates/markets/market_list.html

```diff
@@ -26,10 +26,16 @@
       <th>Ending date</th>
     </tr>
     {% for market in markets %}
-      <tr class="{% if not market.has_started %}table-warning{% elif not market.has_ended %}table-success{% endif %}">
-        <td>
-          <a href="{{ market.get_absolute_url }}">{{ market.title }}</a>
-        </td>
+      <tr class="{% if market.is_upcoming %}table-primary{% elif not market.has_started %}table-warning{% elif not market.has_ended %}table-success{% endif %}">
```

Comment:
Again I think colors should be flipped here; upcoming is better as yellow imo

maybe make this consistent with the markets template too